src/handlers/teacherHomework.ts

```diff
@@ -0,0 +1,54 @@
+// GET
+import { CustomDynamoDB } from "../dynamodb/database";
+import responseHelper from "../helpers/responseHelper";
+import { validateToken } from "../helpers/validations";
+
+const homeworkDB = new CustomDynamoDB('local-bootcamp-classroom-api-homeworks', 'id')
+const homeworkSubmissionDB = new CustomDynamoDB('local-bootcamp-classroom-api-homeworkSubmissions', 'id')
+const userDB = new CustomDynamoDB('local-bootcamp-classroom-api-users', 'email')
+
+export const handler = async(event: any) => {
+    const tokenData = await validateToken(event.headers.Authorization)
+
+    if(!tokenData){
+        return responseHelper(403, 'Fail Authenticating');
+    }
+ 
+    if(tokenData.userType !== "TEACHER"){
+        return responseHelper(401, 'The user type is not a TEACHER');
+    }
+
+    if(!event.pathParameters || event.pathParameters.homeworkId.trim() === '') {
+        return responseHelper(400, 'Homework ID is required')
+    }
+    
+    const homeworkId = event.pathParameters.homeworkId;
+
+    const [homework, user, homeworkSubmission] = await Promise.all([
+        homeworkDB.getItem(homeworkId),
+        userDB.getItem(tokenData.userEmail),
+        homeworkSubmissionDB.query(homeworkId, null, null, "homeworkId-Index")])
```

Comment:
Just for the looks, its better to put the closing square bracket and parenthesis in a new line.
```
const [homework, user, homeworkSubmission] = await Promise.all([
        homeworkDB.getItem(homeworkId),
        userDB.getItem(tokenData.userEmail),
        homeworkSubmissionDB.query(homeworkId, null, null, "homeworkId-Index")
        ])